package.json

```diff
@@ -3,12 +3,11 @@
   "version": "1.0.0",
```

Comment:
Can we please revert all changes in package.json? Some seem to be out-of-date,others seem to be helpful for tests to run locally only.